appbuilder/services/npm-service.ts

```diff
@@ -93,6 +145,21 @@ export class NpmService implements INpmService {
 		}).future<any>()();
 	}
 
+	public isScopedDependency(dependency: string): boolean {
+		let matches = dependency.match(this.scopedDependencyRegExp);
+
+		return !!(matches && matches[0]);
+	}
+
+	public getScopedDependencyInformation(dependency: string): IScopedDependencyInformation {
+		let matches = dependency.match(this.scopedDependencyRegExp);
+
+		return {
+			name: matches[1],
```

Comment:
I think it is better to check if the dependency is scoped before this method is called because if we check here and return `null` if the dependency is not scoped the caller of this method should check if the result is `null`.